src/ConfidentialLedger/README.md

```diff
@@ -0,0 +1,64 @@
+<!-- region Generated -->
+# Az.ConfidentialLedger
+This directory contains the PowerShell module for the ConfidentialLedger service.
+
+---
+## Status
+[![Az.ConfidentialLedger](https://img.shields.io/powershellgallery/v/Az.ConfidentialLedger.svg?style=flat-square&label=Az.ConfidentialLedger "Az.ConfidentialLedger")](https://www.powershellgallery.com/packages/Az.ConfidentialLedger/)
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.2.3 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.ConfidentialLedger`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+require:
+# readme.azure.noprofile.md is the common configuration file
+  - $(this-folder)/../readme.azure.noprofile.md
+input-file:
+# You need to specify your swagger files here.
+  # - $(repo)/specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json
+# If the swagger has not been put in the repo, you may uncomment the following line and refer to it locally
+  - $(this-folder)/../../../azure-rest-api-specs/specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2021-05-13-preview/confidentialledger.json
```

Comment:
Hi @lynshi , please place the swagger path with a public available link. Here is an example for you: https://github.com/Azure/azure-powershell/blob/03527c95d8ad02611bfa7eca0fbacf1583ece6c0/src/ContainerInstance/README.md?plain=1#L39

Reply:
Sorry, the latest version wasn't available when I was working on this. I've updated the link now.